java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -127,6 +171,21 @@ public void subscribe() throws URISyntaxException, IOException, InterruptedExcep
         }
     }
 
+    @SuppressWarnings({"Regexp"})
+    public static boolean isStringBoolean(String input) {
+        String lowerCaseInput = input.toLowerCase();
+        return lowerCaseInput.equals("true") || lowerCaseInput.equals("false");
+    }
```

Comment:
Thinking more, you are using this method to understand if the string is a boolean or not but you are not interested in the value itself, so you need `true` if the string is "true" or "false", not when it's just "true" as it would happen for `parseBoolean`. So take this method but fix my other comments on the lower case and warning.

Reply:
Done